website/docs/reference/node-selection/graph-operators.md

```diff
@@ -3,13 +3,13 @@ title: "Graph operators"
 ---
 
 ### The "plus" operator
-If placed at the front of the model selector, `+` will select all parents of the selected model and the model itself. If placed at the end of the string, `+` will select all children of the selected model and the model itself.
+If placed at the front of the model selector, `+` will select all ancestors of the selected model and the model itself. If placed at the end of the string, `+` will select all descendants of the selected model and the model itself.
```

Comment:
@dbeatty10 should we change this language everywhere in our docs and will this cause confusion with the `parent_depth` and `child_depth` yaml configs?